interface/src/app/node/node.spec.js

```diff
@@ -0,0 +1,246 @@
+/**
+ * 'adage.node' module unit tests
+ *
+ * Test <high-weight-genes> directive.
```

Comment:
Since this file tests other directives too, you may want to put this commend line in a separate section right before line 6 (instead of the comments for the whole file).